src/components/manage/Widgets/SelectWidget.jsx

```diff
@@ -96,6 +96,9 @@ function getDefaultValues(choices, value) {
       value: 'no-value',
     };
   }
+  if (value === null) {
```

Comment:
There's no need for this code, same is achieved if you remove the else return {} at the end on this function.

Reply:
Done!